language/en/interface.json

```diff
@@ -219,8 +219,9 @@
 			"onoff_tooltip": "Active state: turn a unit on/off",
 			"Fighters": "Fighters",
 			"Bombers": "Bombers",
+			"Vtols": "Vtols",
 			"No priority": "No priority",
-			"setpriority_tooltip": "Set target priority Fighters/Bombers/no priority",
+			"setpriority_tooltip": "Set target priority Fighters/Bombers/Vtols/no priority",
```

Comment:
I would not update this, as we are going to remove the command and just have the priorities hardcoded. As well, "Vtols" is not meaningful to the player.

Reply:
@WatchTheFort its all fine since cmd been hidden , so it doesnt matter, as tooltip wont be displayed.

Reply:
Any time an entry in one the translation files changes, it gets flagged by Transifex for re-translation

Reply:
Ok, will remember. Ty